src/containers/Application/ApplicationPage.tsx

```diff
@@ -89,7 +89,7 @@ const ApplicationPage: React.FC<ApplicationProps> = ({
                 sectionCode === strictSectionPage?.sectionCode &&
                 pageNumber === strictSectionPage?.pageNumber
               }
-              canEdit
+              canEdit={current?.status === ApplicationStatus.Draft}
```

Comment:
This was previously being done higher up and passed down in props. Any particular reason for putting it here?

Reply:
Done: 3055394

Reply:
Okay, I think that works. I'm slightly wary about the default being `true` though, as you really wouldn't want something to be editable when it shouldn't be (whereas the reverse would be not as bad).

Just have to ensure it's **always** passed down from Summary and Review. Can you not make it default to `false` and then pass it from ApplicationPage? I don't know, up to you. Moving on...

Reply:
Yes, that makes sense, we don't want that. Well. In this case I will revert this to only pass the canEdit flag and remove the default